gatsby-config.js

```diff
@@ -132,7 +132,7 @@ module.exports = {
     {
       resolve: `gatsby-plugin-sitemap`,
       options: {
-        exclude: ['/imprint', '/data-privacy', '/blog/**'],
+        excludes: ['/imprint', '/data-privacy'],
```

Comment:
oh did that never work as we used the wrong keyword?

Reply:
api changed due to the dependency update

Reply:
test is now fixed, pipeline is green. thanks for the review :)

Reply:
I can't see the blog posts in this file though? (same as on localhost by the way).
I thought it's about the individual posts?
![Screenshot 2021-06-15 at 16 27 09](https://user-images.githubusercontent.com/1701755/122070827-91669480-cdf6-11eb-80d0-a18b0ce1e2a9.png)

Reply:
Oh, nice catch, didn't notice it.

The new sitemap plugin has a new path for the sitemap, it's `sitemap/sitemap-index.xml`. I updated the URL to our `robots.txt`. Maybe you can also do a double check in the Google Search Console and update the path to the sitemap there. Full path should be `https://satellytes.com/sitemap/sitemap-index.xml` after the merge.

Reply:
clarified and answered 👍